Taskfile.yml

```diff
@@ -5,11 +5,16 @@ output: prefixed
 vars:
   GENERATOR_APP: aso-gen
   GENERATOR_ROOT: ./hack/generator/
+  GENERATOR_PACKAGE: github.com/Azure/azure-service-operator/hack/generator
 
   CONTROLLER_APP: aso-controller
   CONTROLLER_ROOT: ./hack/generated/
+  CONTROLLER_PACKAGE: github.com/Azure/azure-service-operator/hack/generated
 
-  CONTROLLER_DOCKER_IMAGE: azure-service-operator:v2.0.0-alpha
+  # Once the v2 controller is the main controller we can drive this from git tags
+  # via e.g. `git describe --tags --dirty` which would give better info for dev builds
+  VERSION: v2.0.0-alpha
```

Comment:
`VERSION` and `SHORT_VERSION` are also set in the `generator:build` step already... is that a problem?

Reply:
Removed `VERSION` which `generator:build` didn’t use, and added a note to this comment to update `generator` once this is the primary version for the code. At the moment we really just want it in the controller’s `User-Agent` for metrics purposes.